Doc/library/stdtypes.rst

```diff
@@ -624,6 +624,23 @@ Additional Methods on Float
 The float type implements the :class:`numbers.Real` :term:`abstract base
 class`. float also has the following additional methods.
 
+.. classmethod:: float.from_number(x)
+
+   Class method to return a floating point number constructed from a number *x*.
```

Comment:
Super-nitpick: I prefer the spelling `floating-point number`, with a hyphen (the "floating-point" part acts as a compound adjective). I think that's what we _mostly_ use throughout the docs (though no doubt there are exceptions).

Reply:
This descriptions was simply copied from the `float` constructor description.

"floating-point" occurs 118 time, "floating point" occurs at least 159 times (in several cases it is split between lines, so it is not easy to get accurate number).

So for now I leave "floating point" for consistency with the constructor description. We will solve this in a separate issue.